test/integration/sign-up-with-id-test.js

```diff
@@ -15,23 +16,29 @@ var options = {
 
 test('sign up with id', function (t) {
   store.clear()
-  t.plan(5)
+  t.plan(6)
 
   var account = new Account({
     url: baseURL,
     id: 'abc4567'
   })
 
+  // mocks
+  var clock = lolex.install(0)
   var mock = nock(baseURL)
     .put('/session/account', function (body) {
+      console.log(body)
```

Comment:
Oh no. That's bad, sorry I was using it check the response earlier. I'll get rid of it right now.